app/controllers/service_worker_controller.rb

```diff
@@ -6,9 +6,4 @@ def index
     set_surrogate_key_header "serviceworker-js"
     render formats: [:js]
   end
```

Comment:
Do we need this controller at all anymore, or is it used for 'deleting' existing service workers?

Reply:
We need it still for the self destructing service worker.